tests/report_test.py

```diff
@@ -77,7 +77,9 @@ def _assert_empty(changed_files, postfix):
 
 
 def _assert_empty_json(changed_files, postfix=".json"):
-    """ Asserts there are no changed files and otherwise builds error message"""
+    """Asserts there are no changed files and otherwise builds error
+    message."""
+    postfix = ".json"
```

Comment:
You shouldn't overwrite the postfix here, I think that changes some test behaviour